conda/recipes/cuml-cuda10/meta.yaml

```diff
@@ -32,12 +32,14 @@ requirements:
     - libcuml={{ version }}
     - libcumlMG
     - cudatoolkit {{ cuda_version }}.*
+    - cupy 6.0.*
```

Comment:
We cannot merge this until CuPy conflicts with nccl are solved

Reply:
Ok, I got you. Please let me know when a newer version of cupy works (like 7.0.0 which has just been released), and I will update the conda recipe.